### ISSUE TITLE
In-Call Chat: Close button contrast

### ISSUE THREAD

**Issue by neeti-s** [berkmancenter/frankly#574]
Close button contrast is low and does not meet accessibility standards. The X close button must have its color updated to Primary (standard hex is #313030)

<img width="921" height="2048" alt="Image" src="https://github.com/user-attachments/assets/d8cb0d49-a81c-41df-80e5-6ad64be0ac9e" />

Tasks:
1. Update close X with Primary color

**Comment by mwmarino** [comment]
Converting this to a task (unclear if this is actually a bug, versus an oversight on button color and contrast)
